zubhub_frontend/zubhub/src/api/api.js

```diff
@@ -277,13 +277,21 @@ class API {
    */
   removeTeamMember = ({ groupname, username, token }) => {
     const url = `creators/${groupname}/remove-member/${username}/`;
-    const method = 'DELETE';
+    const method = 'GET';
     if (token) {
-      return this.request({ url, method ,token }).then(res => res.json());
+      return this.request({ url, method ,token }).then(res =>  Promise.resolve(res.status === 204 ? { detail: 'ok' } : res.json()))
     } else {
       return this.request({ url, method }).then(res => res.json());
     }
   };
+  
+  deleteAccount = ({ token }) => {
```

Comment:
This one

Reply:
I see i was trying to figure out how it worked it was written by @NdibeRaymond  i have remove it just now